apps/front-end/src/components/popup/leftPane/LeftPane.tsx

```diff
@@ -10,7 +10,7 @@ import { renderIfData } from "../../../utils/jsx-utils";
 
 interface LeftPaneProps {
   name: string;
-  food_system_category: string[];
+  food_system_category?: string;
```

Comment:
Likewise, big warning comment saying this is very Powys-specific and should be unhardwired later would be good.